Fixed formatting issues with calendar events

YongJunLim opened the pull request:
Added missing spaces/brackets, fixed truncated names

Examples:
SUMMARY:Science for a Sustainable Worl --> SUMMARY:Science for a Sustainable World
SUMMARY:Global Humaniti and DESCRIPTION:Global Humanities:Lit\,Philo\,Et --> SUMMARY:Global Humanities 
SUMMARY:Digital Worlds\, Space and Spat --> SUMMARY:Digital Worlds, Space and Spatialities
LOCATION:Albert HongLec Theatre (1.102 --> LOCATION:Albert Hong Lec Theatre (1.102)
LOCATION:Cohort Classroom2 (1.315) --> LOCATION:Cohort Classroom 2 (1.315)
LOCATION:Think Tank2(1.309) --> LOCATION:Think Tank 2 (1.309)

YongJunLim committed:
Fixed formatting issues with calendar events

Added missing spaces/brackets, fixed truncated names

Examples:
SUMMARY:Science for a Sustainable Worl --> SUMMARY:Science for a Sustainable World
SUMMARY:Global Humaniti and DESCRIPTION:Global Humanities:Lit\,Philo\,Et --> SUMMARY:Global Humanities
SUMMARY:Digital Worlds\, Space and Spat --> SUMMARY:Digital Worlds, Space and Spatialities
LOCATION:Albert HongLec Theatre (1.102 --> LOCATION:Albert Hong Lec Theatre (1.102)
LOCATION:Cohort Classroom2 (1.315) --> LOCATION:Cohort Classroom 2 (1.315)
LOCATION:Think Tank2(1.309) --> LOCATION:Think Tank 2 (1.309)

MarkHershey commented on tests/resources/ics/term8_new.ics:
```diff
@@ -50,560 +50,560 @@ EXDATE;TZID=Asia/Singapore:20220710T083000,20220809T083000
 BEGIN:VEVENT
 SUMMARY:Artificial Intelligence
-LOCATION:Think Tank 4 (1.312)
+LOCATION:Think Tank 4 (1.312))
```
Seems like an extra closing parenthesis is added?

MarkHershey reviewed (changes requested):
Please help fix the parenthesis issue, the rest looks good to me. Thanks a lot!

YongJunLim committed:
Fixed double bracket errors

Also added more misspelled module names to spot and replace

YongJunLim commented:
I fixed the error where an extra closing parenthesis is accidentally added and added more misspelled module names to replace. I also rebased my changes onto your new commits from yesterday.

MarkHershey commented on tests/resources/ics_outputs/term8.ics:
```diff
@@ -50,560 +50,560 @@ EXDATE;TZID=Asia/Singapore:20220710T083000,20220809T083000
 BEGIN:VEVENT
 SUMMARY:Graphics & Visualization
-LOCATION:Think Tank 2(1.309)
+LOCATION:Think Tank 2 (1.309))
```
Something is still not right, there are double closing bracket for all files.

MarkHershey commented on tests/resources/ics_inputs/2023_term1_new.ics:
remove the output files `*_new.ics` from `ics_inputs/` dir and re-generate the outputs by running `pytest`

YongJunLim replied:
Oh I tried running pytest after seeing this message and did not find the double closing bracket in the output files

MarkHershey reviewed (changes requested):
Thanks! Please check again.

YongJunLim committed:
ran pytest and organised test ics files

MarkHershey reviewed (approved):
Thanks for running the tests, it seems all good now, thanks again for contributing!
